packages/bpk-docs/src/pages/DrawerPage/DrawerPage.js

```diff
@@ -18,7 +18,7 @@
 
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import BpkSlidingDrawer from 'bpk-component-drawer';
```

Comment:
is the Drawer supposed to be part of this PR?

Reply:
No it was a JDI ... 😇